src/main/js/distribution/job/HipChatJobConfiguration.js

```diff
@@ -99,6 +99,7 @@ class HipChatJobConfiguration extends Component {
                 <TextInput
                     id={KEY_ROOM_ID}
                     label="Room Id"
+                    description="The ID of the room to receive Alerts."
```

Comment:
Might want to change this to say, "The API ID of rht room to receive alerts."

It's officially called API ID in HipChat docs.